### PR TITLE
Bug fix for dump when data directoryis not exist

### LINKED ISSUE #1008: gitea dump fails on 1.0.2
- Gitea version (or commit ref): 1.0.2
- Git version: 1.9.1
- Operating system: Ubuntu 14.04.5 LTS (trusty)
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No
  - [ ] Not relevant
- Log gist:

https://gist.github.com/jbub/613738c173c6f6e9c78ab95e8251b535

## Description

I can no longer run `gitea dump` after upgrading to 1.0.2. Please see the log for more info.

### PR DESCRIPTION
**Pull request by lunny**
fix #1008

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

File: cmd/dump.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/setting"
 	"github.com/Unknwon/cae/zip"
+	"github.com/Unknwon/com"
```
Comment (strk) [inline cmd/dump.go:20]: is there no simple standard way to check if a directory exists ? I'd try to keep number of dependencies down...

**Review by strk — COMMENTED** [review]
Untested, but shouldn't check for existance happen *after* building the full absolute path ?

File: cmd/dump.go (hunk 1 above)
Comment (lunny) [reply cmd/dump.go:20]: Since `github.com/Unknwon/com` has been imported many place, I just used it. Or we can write a function about 10 lines.

**Comment by lunny** [comment]
@strk any new idea?

**Comment by strk** [comment]
No ideas, LGTM :)

**Pull request merged**